.github/workflows/ci.yml

```
@@ -52,9 +53,11 @@ jobs:
 
   install-opam:
     strategy:
+      fail-fast: false
       matrix:
         os: [ubuntu-latest]
         ocaml-compiler: [4.13.x]
+        coq: [8.18.0, 8.19+rc1]
```

```suggestion
        coq: [8.18.0, 8.19+rc1, dev]
```

This is probably enough @rtetley

---

also, we can use 8.19.0 today

---

Sounds good !